deploy/charts/rook-ceph-cluster/templates/deployment.yaml

```diff
@@ -7,6 +7,9 @@ metadata:
   namespace: {{ .Release.Namespace }} # namespace:cluster
   labels:
     app: rook-ceph-tools
+  {{- if .Values.toolbox.labels }}
```

Comment:
The indentation is visually different, can you increase the indentation of these lines to match the `app` label?